config/environment.rb

```diff
@@ -2,4 +2,7 @@
 require_relative 'application'
 
 # Initialize the Rails application.
-Rails.application.initialize!
+begin
+  Rails.application.initialize!
+rescue ActiveRecord::ConnectionNotEstablished
```

Comment:
Any idea why this is happening? I'm concerned that this might be masking a real issue somewhere else... I'd at least like a comment saying why this is here...